executor/builder.go

```diff
@@ -4665,18 +4630,30 @@ func (b *executorBuilder) buildBatchPointGet(plan *plannercore.BatchPointGetPlan
 		partTblID:        plan.PartTblID,
 		columns:          plan.Columns,
 	}
-	if plan.TblInfo.TableCacheStatusType == model.TableCacheStatusEnable {
-		e.cacheTable = b.getCacheTable(plan.TblInfo, startTS)
-	}
+
 	if plan.TblInfo.TempTableType != model.TempTableNone {
 		// Temporary table should not do any lock operations
 		e.lock = false
 		e.waitTime = 0
 	}
 
+	txnManager := sessiontxn.GetTxnManager(b.ctx)
+	var err error
 	if e.lock {
```

Comment:
Why the code is different with `PointGet`, I think they can also use `b.getSnapshotTS` instead of `xnManager.GetStmtForUpdateTS`

Reply:
I modify the point get and forget to change this accordingly. I have changed it now.